sanic/sanic.py

```diff
@@ -273,10 +273,15 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
         :param protocol: Subclass of asyncio protocol class
         :return: Nothing
         """
-        logging.basicConfig(
-            level=logging.INFO,
-            format="%(asctime)s: %(levelname)s: %(message)s"
-        )
+        # Only set up a default log handler if the
+        # end-user application didn't set anything up.
+        if not logging.root.handlers and log.level == logging.NOTSET:
```

Comment:
Should this also replace the settings in serve_multiple? https://github.com/awiddersheim/sanic/blob/b459f9c8ed6b459d8c7328bf02746e0f8e6a395b/sanic/sanic.py#L374